DANS-KNAW/dd-verify-migration
# DD-1488 upgrade parent

Fixes DD-1488 upgrade parent

# Description of changes

# How to test

- Fresh `dev_archaeology-2023-12-07.box` 
- `poetry run dv-dataset-verify doi:10.5072/DAR/R4RTPO`
- build,  deploy, check in VM: `dd-verify-dataset --version` 0.12.1-SNAPSHOT
- repeat step two with identical results

# Related PRs

(Add links)

*

# Notify

@DANS-KNAW/dataversedans
